qiita_plugins/target_gene/tgp/util.py

```diff
@@ -0,0 +1,187 @@
+# -----------------------------------------------------------------------------
```

Comment:
OK, that makes sense. BTW while you work on the unifrac plugin it will be great to write a tutorial of how to write them. Perhaps start by copying this file and modifying it.